java/core/src/main/java/co/worklytics/psoxy/rules/Rules2.java

```diff
@@ -117,22 +117,4 @@ public Rules2 clone() {
             .build();
         return clone;
     }
-
-    //TODO: fix YAML serialization with something like
-    // https://stackoverflow.com/questions/55878770/how-to-use-jsonsubtypes-for-polymorphic-type-handling-with-jackson-yaml-mapper
-
-    static ObjectMapper mapper = new YAMLMapper();
-
-    public static synchronized Rules2 load(String path) {
-        try {
-            URL url = PrebuiltSanitizerRules.class.getClassLoader().getResource(path);
-            if (url == null) {
-                throw new IllegalArgumentException("No such file: " + path);
-            }
-            return mapper.readerFor(Rules2.class).readValue(Files.readAllBytes(Paths.get(url.toURI())));
-        } catch (IOException | URISyntaxException e) {
-            throw new Error(e);
-        }
-    }
```

Comment:
this is not really slack specific ... why not have in `Rules2?

Reply:
Well, it wasn't used in any other place after fixing Slack thing.